.github/ISSUE_TEMPLATE/questions.md

```diff
@@ -0,0 +1,7 @@
+---
+name: "❓ Questions"
+about: Please use https://forum.serverless.com, StackOverflow or other forums for questions
+---
+
+Please use https://forum.serverless.com, StackOverflow or other forums for questions.
```

Comment:
Can we suggest some specific StackOverflow labels users should mark their questions with?

Reply:
Added in https://github.com/serverless/serverless/pull/6786/commits/fa22add122cceccc9f1ca03f9fc3991d0393686f